packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -282,22 +264,96 @@ describe("<DatePicker /> - Input format", () => {
 })
 
 describe("<DatePicker /> - Validation", () => {
+  it("displays the message when status is error", () => {
```

Comment:
```suggestion
  it("displays custom validation message when provided (overrides inbuilt validation)", () => {
```

Hmm... actually we may need to adjust the props to make it meet an inbuilt validation error, which we ensure isn't shown.